avahi-sharp/Makefile.am

```diff
@@ -17,7 +17,8 @@
 
 ASSEMBLY = avahi-sharp.dll
 
-CLEANFILES = $(ASSEMBLY) $(ASSEMBLY).mdb $(ASSEMBLY).config
+CLEANFILES = $(ASSEMBLY) $(ASSEMBLY).mdb $(ASSEMBLY).config \
+	avahi-sharp-docs.zip avahi-sharp-docs.tree
```

Comment:
As far as I can see those things are generated in the `HAVE_MONODOC` part. Can you append them to `CLEANFILES` there?